rpcs3/Emu/SysCalls/Modules/cellSail.h

```diff
@@ -610,74 +664,83 @@ struct CellSailSourceStreamingProfile
 
 union CellSailEvent
 {
-	struct u32x2 {
+	struct _data_t
+	{
 		be_t<u32> major;
 		be_t<u32> minor;
-	};
+	} u32x2;
```

Comment:
I'm not sure, do you really want to write "u32x2." every time you access major/minor?